WPMedia/fusion-news-theme-blocks
# TMEDIA-99 - Open navigation menu fixes, z-index and width

## Description

Add z-index to nav element to ensure any children elements do not show on top of the open sections menu
Fixs the width being 200% when open and causing horizontal scrolling

## Jira Ticket
- [TMEDIA-99](https://arcpublishing.atlassian.net/browse/TMEDIA-99)

## Test Steps

1. Checkout this branch `git checkout TMEDIA-99-header-nav-menu-z-index-when-open`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/header-nav-chain-block,@wpmedia/shared-styles`
3. Grab latest data from core components or add an HTML element with the the following code to make your features look like - 

```<div style="background-color: rgb(70 146 199); padding: 1rem; position: relative; color: white; font-weight: bold">Custom Something</div>```

<img width="250" alt="TMEDIA-99-features" src="https://user-images.githubusercontent.com/868127/114578436-c2024480-9c74-11eb-9f2e-339d512e3b41.png">

4. Visit the published page - http://localhost/homepage/?_website=the-gazette
5. Verify the new HTML element added sits below sections menu when opened

## Effect Of Changes
### Before

Element with position relative sits over open sections menu

<img width="1794" alt="TMEDIA-99-before" src="https://user-images.githubusercontent.com/868127/114578072-7e0f3f80-9c74-11eb-8fbe-04a2c3bc1788.png">


### After

Sections menu sits on top of child elements

<img width="1792" alt="TMEDIA-99-after" src="https://user-images.githubusercontent.com/868127/114578129-8cf5f200-9c74-11eb-9917-738ee2fb38fe.png">


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
